inferno/lib/rule.py

```diff
@@ -29,6 +30,39 @@ def crc_partition(key, nr_partitions, params):
     import binascii
     return binascii.crc32(key) % nr_partitions
 
+'''
+    The dateLookBack function will take in a list of sourceTags and the number of days 
+    you want to return on the sourceTag list. For example, if today is October 14, 2015 
+    and you enter the following:
+    dateLookBack('incoming:collector:chunk:searches', 3) will return a list containing
+    ['incoming:collector:chunk:data_channel_params:2015-10-14', 'incoming:collector:chunk:data_channel_params:2015-10-13'
+    'incoming:collector:chunk:data_channel_params:2015-10-12', 'incoming:collector:chunk:data_channel_params:2015-10-11']
+
+    arguments:
+        sourceTagList: The list of the sourceTags you want to use the rule on
+        lookBackNumber: An integer number of days you want to look look back 
+
+    Returns: A list containing all of the sourceTags with the date appended to them
+
+'''
+
+def dateLookBack(sourceTagList, lookBackNumber):
```

Comment:
Rename to something pythonic eg: `get_date_lookback` and move to bottom of the file. Also, move the docstring into the function and use `"""` instead of `'''`